app/Enums/BackupStatus.php

```diff
@@ -32,6 +32,6 @@ public function getColor(): string
 
     public function getLabel(): string
     {
-        return str($this->value)->headline();
+        return trans('strings.backupstatus.'.str($this->value));
```

Comment:
```suggestion
        return trans('strings.backupstatus.' . $this->value);
```
No need for str when not using headline since this Enum is a string
https://github.com/pelican-dev/panel/blob/de166bca0319f1240509f7927d2d26abb20a3b16/app/Enums/BackupStatus.php#L9